src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -215,6 +224,18 @@ public double getCosineIntakeAngle() {
 
   @Override
   public void periodic() {
+    if (desiredDeployed) {
+      if (clearOfIntake.getAsBoolean()) {
+        deploy();
+      }
+    }
+
+    if (desiredDeployed == false) {
+      if (clearOfIntake.getAsBoolean()) {
+        retract();
+      }
+    }
```

Comment:
I think you can restructure this a little bit just to be more concise
```suggestion
    if (clearOfIntake.getAsBoolean()) {
      if (desiredDeployed) {
        deploy();
      }
      else {
        retract();
      }
    }
```

Does the same thing but it's just shorter